includes/rest/class-outbox-controller.php

```diff
@@ -233,10 +233,14 @@ public function get_items( $request ) {
 	 * @return array Response object on success, or WP_Error object on failure.
 	 */
 	public function prepare_item_for_response( $item, $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
```suggestion
	public function prepare_item_for_response( $item, $request ) {
```
Looks like `$request` is now used